astropy/units/format/latex.py

```diff
@@ -73,8 +93,10 @@ class LatexInline(Latex):
     <https://journals.aas.org/manuscript-preparation/>`_.
     """
 
-    name = "latex_inline"
+    name: ClassVar[str] = "latex_inline"
 
     @classmethod
-    def to_string(cls, unit, fraction=False):
+    def to_string(
+        cls, unit: UnitBase, fraction: bool | Literal["inline", "multiline"] = False
+    ) -> str:
         return super().to_string(unit, fraction=fraction)
```

Comment:
Isn't this override actually unnecessary ?

Reply:
It's needed because it changes the default value of `fraction`.